app/controllers/deploys_controller.rb

```diff
@@ -139,6 +134,6 @@ def find_deploy
   end
 
   def active_deploy_scope
-    @project ? @project.deploys.active : Deploy.active
+    current_project ? current_project.deploys.active : Deploy.active
```

Comment:
Calling `current_project` here to know if it exists will call `require_project`, which will throw an error if there's no project.  I think in this case, you want to leave it as @project.